cytoscape-autopan-on-drag.js

```diff
@@ -184,7 +184,15 @@
       // mark that the extension is disabled now
       setScratch(cy, 'enabled', undefined);
     }
-    
+
+    function getWidth (cy) {
+      return cy.container().clientWidth;
```

Comment:
jQuery's `width` method does not include the padding. Whereas the `clientWidth` property includes the padding. Therefore you cannot replace one with another! Same thing applies to the height.